Documentation/SettingUpYourXcodeProject.md

```diff
@@ -1,16 +1,20 @@
 # Setting Up Tests in Your Xcode Project
 
 When you create a new project in Xcode 7, a unit test target is included
-by default. To write unit tests, you'll need to be able to use your main
-target's code from within your test target.
+by default (unless it's a Command Line Tool project). To write unit tests, 
```

Comment:
Could we provide instructions on how to add a unit test target for command line tool projects?